src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -126,14 +125,60 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
             echo "gfal-copy exit status: $EXIT_STATUS"
             if [[ $EXIT_STATUS != 0 ]]; then
                echo "ERROR: gfal-copy exited with $EXIT_STATUS"
+               echo "Actual command which failed: %s"
```

Comment:
Andrea, perhaps we could print the actual command to be execute outside of this bash function. Somewhere inside this method with the `logging` module.

Reply:
Ok, I will provide an implementation of this idea with the next commit